nx/lib/nx/binary_backend/matrix.ex

```diff
@@ -377,16 +381,19 @@ defmodule Nx.BinaryBackend.Matrix do
             if is_nil(q) do
               h
             else
-              dot_matrix(q, h)
+              multiply_matrix(q, h)
             end
 
           # Hessenberg matrix H updating
-          h_t = transpose_matrix(h)
+          h_cc =
+            h
+            |> transpose_matrix()
+            |> Enum.map(fn a_row -> Enum.map(a_row, &Complex.conjugate(&1)) end)
```

Comment:
I would write an `ajdoint_matrix` function that transposes and conjugates in a single pass

Reply:
I did not know what to do. Do you mean that I should define a private function `adjoint_matrix` ?

Reply:
yes

Reply:
I defined private function `adjoint_matrix`.